protocol-specs/generator_outputs/lisk_codec/validBytesEncodings.json

```diff
@@ -0,0 +1,51 @@
+{
+	"title": "Valid bytes encodings",
+	"summary": "Examples of encoding bytes as required by lisk-codec",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "lisk_codec",
+	"handler": "validBytesEncodings",
+	"testCases": [
+		{
+			"description": "Encoding of bytes types",
+			"config": {
+				"network": "devnet"
+			},
+			"input": {
+				"bytes": {
+					"object": {
+						"address": "á\u001a\u00116G8\"X\u0013øn¨R\u0014@\u000e]°n"
```

Comment:
why cant we pass a Buffer array (like you do in the other file later on), instead of this encoded string?

Reply:
This was changed due to previous feedback. This whole file is using binary string representation now.